internal/builders/dockerfile/build.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2022 SLSA Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/slsa-framework/slsa-github-generator/internal/builders/dockerfile/docker"
+)
+
+// buildCmd returns the 'build' command.
+func buildCmd() *cobra.Command {
```

Comment:
This is mostly just a thin wrapper over the `docker build` command currently. Need to think a bit on what advantage wrapping in a Go app has. Maybe just a bit more control over what command is run?